### PR TITLE
FCCEUD-79 Updates to pantheon2.yml file configuration descriptions

### PR DESCRIPTION
**Pull request by stoobie**
Based on emarcusRH:fcceud-79

### CODE REVIEW

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc [hunk 1]
```diff
@@ -16,18 +15,55 @@ The configuration file enables you to import content from a Git repository into
...
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
-** `modules` (optional) Patterns for modules that show up in the search results on {ProductShortName}.
-** `resources` (optional) Image files, legal information, or other content that is not a complete module or assembly.
-** `attributes` (optional) A path to the repository-wide attributes file. Provide this value if your documents relies on the attributes file.
-+
-NOTE: You must define the values for `server` and `repository`.
+`\https://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
+** `path` (Optional) The path containing resource files, such as `attributes.adoc`.
```
Comment (noopurmath3) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:26]: `path` is not an optional filed, it is required as per the recent changes

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc [hunk 2]
```diff
@@ -16,18 +15,55 @@ The configuration file enables you to import content from a Git repository into
...
+** `name` (Optional) A unique ID that is displayed in URLs on the {ProductShortName} site when you view modules and assemblies. If no value is provided for `name`, this field is defined as *DEFAULT*.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
-`\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
-** `modules` (optional) Patterns for modules that show up in the search results on {ProductShortName}.
-** `resources` (optional) Image files, legal information, or other content that is not a complete module or assembly.
-** `attributes` (optional) A path to the repository-wide attributes file. Provide this value if your documents relies on the attributes file.
-+
-NOTE: You must define the values for `server` and `repository`.
+`\https://<_{ProductShortName}__url>/content/repositories/*repository*/entities/modules/getting_started/foo.adoc.preview`
```
[the remark this answers is not in the document]
Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:25]: How's this:
```
For example, if you specify `repository: rhel-docs`, `name: rhel8`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
`\https://<_{ProductShortName}__url>/pantheon/#/module/repositories/rhel-docs/entities/foo.adoc?variant=rhel8`
```

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc (hunk 1 above)
Comment (stoobie) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:26]: Fixed.

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (noopurmath3) [reply pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc:25]: Looks good

**Review by noopurmath3 — APPROVED** [review]
LGTM 👍

**Pull request merged**